src/librustdoc/html/render/mod.rs

```diff
@@ -1691,7 +1708,7 @@ pub(crate) fn render_impl_summary(
         format!(" data-aliases=\"{}\"", aliases.join(","))
     };
     write!(w, "<div id=\"{}\" class=\"impl has-srclink\"{}>", id, aliases);
-    render_rightside(w, cx, &i.impl_item, containing_item);
+    render_rightside(w, cx, &i.impl_item, containing_item, RenderMode::Normal);
```

Comment:
Are we sure that this code is never used for rendering methods from Deref?

Reply:
This is for the impl summary (so before all the methods). So I don't expect it to be an issue.

Reply:
Makes sense.

Reply:
Hum actually, the function is called after [this check](https://github.com/rust-lang/rust/blob/master/src/librustdoc/html/render/mod.rs#L1581). Are you really sure you want me to add an extra argument to the function anyway?

Reply:
Did you comment on the wrong thread? I'm confused about what you mean.

Reply:
I completely did, so it's why my answer doesn't make any sense. I was answering to another comment on the bad code...